tests/test_telegramobject.py

```diff
@@ -284,3 +284,45 @@ def test_deepcopy_subclass_telegram_obj(self, bot):
         assert d._private == s._private  # Can't test for identity since two equal strings is True
         assert d._bot == s._bot and d._bot is s._bot
         assert d.normal == s.normal
+
+    def test_string_representation(self):
+        class TGO(TelegramObject):
+            def __init__(self, api_kwargs=None):
+                super().__init__(api_kwargs=api_kwargs)
+                self.string_attr = "string"
+                self.int_attr = 42
+                self.to_attr = BotCommand("command", "description")
+                self.list_attr = [
+                    BotCommand("command_1", "description_1"),
+                    BotCommand("command_2", "description_2"),
+                ]
+                self.dict_attr = {
+                    BotCommand("command_1", "description_1"): BotCommand(
+                        "command_2", "description_2"
+                    )
+                }
+                # Should not be included in string representation
+                self.none_attr = None
+
+        expected_without_api_kwargs = (
+            "TGO(dict_attr={BotCommand(command='command_1', description='description_1'): "
+            "BotCommand(command='command_2', description='description_2')}, int_attr=42, "
+            "list_attr=[BotCommand(command='command_1', description='description_1'), "
+            "BotCommand(command='command_2', description='description_2')], "
+            "string_attr='string', to_attr=BotCommand(command='command', "
+            "description='description'))"
+        )
+        assert str(TGO()) == expected_without_api_kwargs
+        assert repr(TGO()) == expected_without_api_kwargs
```

Comment:
```suggestion
        assert str(TGO()) == repr(TGO()) == expected_without_api_kwargs
```
single line is better?

Reply:
wouldn't that make it harder to see which of the two assertions fails, if one fails?

Reply:
py 3.10+ shows where the assertion fails with carets, so it's okay to do this I guess

Reply:
mh, I'll keep it in two lines for now … py3.9 will reach EOL only in years